masterlist.yaml

```diff
@@ -136,3 +136,75 @@ plugins:
     url: [ 'http://www.nexusmods.com/fallout4/mods/2142' ]
     after:
       - 'Simple Bug Fixes.esp'
+
+  - name: 'DarkerNights-TrueStorms-LITE-Audio.esp'
+    url: [ 'http://www.nexusmods.com/fallout4/mods/191' ]
+    after:
+      # Place DarkerNights-TrueStorms-LITE-Audio.esp after TrueStormsFO4-LITE-Audio.esp
+      # to ensure the magic of Darker Nights is applied to WTHR records modified by the True Storms LITE Audio addon
+      - 'TrueStormsFO4-LITE-Audio.esp'
+    msg:
+      # Display a warning when Darker Nights patch for True Storms LITE Audio addon is NOT used along with the True Storms LITE Audio addon
+      - type: warn
+        content: 'When plugin ''TrueStormsFO4-LITE-Audio.esp'' is **NOT** active plugin ''DarkerNights-TrueStorms.esp'' should be used instead. TIP: Reinstall the **Darker Nights for True Storms** patch and select the option which matches your True Storms mod installation.'
```

Comment:
I do not like how you're wording these messages:

1. The message is already conditional, why are you including the condition in the message too? All that will do is make users think it's not conditional and that they must check for themselves, which is self-defeating.
2. You state what is wrong, then provide the solution as a tip? It's not a tip, it's an instruction.

One way this could be better written: `This plugin does not match the active True Storms plugin. Reinstall the **Darker Nights for True Storms** patch and select the option which matches your True Storms mod installation.`